### DISCUSSION TITLE
[BUG]After creating a volume through Longhorn UI and creating PV/PVC, it is not automatically attached

### DISCUSSION THREAD

**Discussion by w7team** [longhorn/longhorn#9725] [General]
通过kubectl操作k8s接口创建pv/pvc会自动attach，但是通过Longhorn UI操作的话，创建volume并且创建PV/PVC，不会自动attach，上次discuss问了官方好像也没解释清楚为啥会这样。手动操作attach只能选择1个节点，我有多个节点多个副本的情况下也不知道选择哪个对，主要还是不知道attach到某个节点上，这个节点是起到什么作用，修改到其他节点会有什么影响，一般自动attach是怎么选择的？

Using kubectl to operate the k8s interface to create PV/PVC will automatically attach, but when operating through the Longhorn UI, creating a volume and then PV/PVC does not result in automatic attachment. Last time in the discussion, I asked the official team, but they didn't seem to explain clearly why this happens. When manually attaching, you can only select one node, and in the case of multiple nodes and replicas, I'm not sure which one to choose. Mainly, I don't understand what role the node plays after attaching to it, or what impact there would be if I switch it to another node. How is the automatic attachment usually determined?

![image](https://github.com/user-attachments/assets/d7cd94fc-6201-4c74-b140-aa860b34431d)

**Comment by derekbit** [comment]
@w7team As I mentioned before, your questions can be found in the [official document](https://longhorn.io/). :)
